src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -953,11 +957,15 @@ function updateNumScalePossibleValues(questionNumber) {
         $("#maxScaleBox"+idSuffix).val(max);
     }
     
-    if (step < 0.001) {
+    step = roundToThreeDp(step);
+    if (step == 0) {
         step = 0.001;
-        $("#stepBox"+idSuffix).val(step);
     }
-    
+    $("#stepBox" + idSuffix).val(step);        
+    if (isNaN(step)) {
+        $("#stepBox" + idSuffix).val("");        
```

Comment:
Hmmm if I do that then this line in the test page:
```
feedbackEditPage.fillStepNumScaleBox("", -1);
```
Will result in the box filled with 1 instead of being cleared. Not that it's test-breaking, just counter-intuitive.
Also, this extension is only applicable for the test page. For the user, their input is already limited to numbers only, so it won't ever be NaN.